src/ui/EditTask.svelte

```diff
@@ -295,65 +297,65 @@
                         id="priority-{value}"
                         {value}
                         bind:group={editableTask.priority}
-                        accesskey={label.charAt(0).toLowerCase()}
+                        accesskey={withAccessKeys ? label.charAt(0).toLowerCase() : null}
```

Comment:
I'm not sure whether you want to change it, but I do note that this is quite a repetitive change.
Is it possible to hide the new logic inside some kind of helper function, perhaps?
Totally optional. It is at least readable what is going in with the `accesskey=` lines...

Reply:
Ok, I made it a bit more dry now.